src/plugins/pcr/src/PrimerStatistics.cpp

```diff
@@ -136,7 +136,8 @@ PrimerStatisticsCalculator::PrimerStatisticsCalculator(const QByteArray &sequenc
         case 'N':
             break;
         default:
-            FAIL(QString("Unexpected symbol: ") + c, );
+            initializationError = QString(PrimerStatistics::tr("Unexpected symbol: ") + c);
```

Comment:
`initializationError = PrimerStatistics::tr("Unexpected symbol: %1").arg(c);`

Reply:
I see no reason to use your code instead of mine.